### ISSUE TITLE
NVDA is not announcing the descriptive information of "Example" link under "Select Group" dialog.

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#13965]
### Steps to reproduce:
1.Open File Explorer.
2.Right click on a folder.
3.Navigate to properties.
4.Navigate to security tab.
5.Navigate to edit button.
6.then navigate to add button.
7.Verify the announcement of 'examples' link using NVDA.

### Actual behavior:

NVDA is not announcing the descriptive information of "Example" link under "Select Group" dialog.

### Expected behavior:

"Example" link should be descriptive under "Select Group" dialog.
Solution: Screen reader should announce like "Enter the object name to select example link"

#### NVDA installed/portable/running from source:
Installed
#### Windows version:

Windows 11 , version-21H2 (Build 22000.795)

#### Name and version of other software in use when reproducing the issue:
No

#### Does the issue still occur after restarting your computer?

Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.

https://user-images.githubusercontent.com/93735775/181824642-e7e2ddb6-df4c-4483-a64b-8671670b9892.mp4

**Comment by seanbudd** [comment]
Like other edit fields, NVDA correctly reads the label "Enter the object name to select example" when tabbing to the edit field.

**Comment by feerrenrut** [comment]
NVDA speech when the dialog is opened and read out:
```
Select Users or Groups  dialog  Select this object type:
Users, Groups, or Built-in security principals
From this location:
LAPTOP-LUJTNHBS
examples
Enter the object names to select (<A>examples</A>):  edit  multi line  Alt+e  blank
```

NVDA speech when using tab to focus the link:
```
examples  link
```

Accessibility insights shows no extended information for the "examples" link.
Windows would need to expose an extended description (via IAccessible or UIA) for the screen reader to announce it.
Please have Windows / Explorer engineers look into this.
